components/api/core/api.bal

```diff
@@ -42,11 +42,11 @@ service CelleryHubAPI on ep {
     @http:ResourceConfig {
```

Comment:
Shouldn't this file be moved inside the gen module. As this is completely auto generated ?

Reply:
This may encounter a cyclic import problem.